### ISSUE TITLE
Memorybreakpoint event returns None type on FrameAdvance()

### ISSUE THREAD

**Issue by TryTwo** [Felk/dolphin#19]
The python script is not using frameadvance at all, but still seems to be triggered by it.
```
while True:
        (is_write, addr, value) = await event.memorybreakpoint()
        print(f"{addr:08x} was changed to {value}")
```
An await event.memorybreakpoint() loop will break the script when None type is returned.  The correct way to write the code currently is:

```
while True:
    try:
        (is_write, addr, value) = await event.memorybreakpoint()
    except:
         continue

    print(f"{addr:08x} was changed to {value}")
```

A callback will throw errors, but not break the script.

**Comment by Felk** [comment]
thanks for reporting the issue, I'll take a look when I get back to developing on this fork.

**Comment by TryTwo** [comment]
I tried to solve this myself, but I couldn't get it.  If you get some time could you look into fixing just this?  I'd like to make some custom builds with python, but this error makes writing scripts a bit tricky.

**Comment by Felk** [comment]
I attempted to fix this in 02ce4ecbcb1be256936465dfcb994880eb62da2e

**Comment by TryTwo** [comment]
My quick test had everything working correctly. Thanks for fixing!

I'm not too skilled with python. Is this code the normal way to get the u32 address -> tuple that's just a single int -> int?
```
    address = await event.codebreakpoint() 
    addr = address[0]
```
